Backend/router/v1/report.ctrl.js

```diff
@@ -45,13 +55,51 @@ const deleteReport = (req, res) => {
       id: reportId
     }
   }).then((result) => {
-    res.status(204).json(result);  //no content
-  }).catch((err) => {});
+    if(result == ""){
```

Comment:
I see this if/else block being repeated 3 times here and there. Please create a new function and refactor.

Reply:
@shberry if it is number OR if result is "" ? all of these?

Reply:
I tried to make a new function, but because of the sync, I use promise and I think the code will be longer. @shberry